### PR TITLE
Add Patrick bio

### PR DESCRIPTION
**Pull request by patrick-axiom**
It's me! On the Internet!

### CODE REVIEW

**Review by staceybuckelew — COMMENTED** [review]
Web bio updates look good to me!

File: _data/staff.yml [hunk 1]
```diff
@@ -114,6 +114,12 @@
   image: "c-turner"
   email: "chris@axiomdatascience.com"
 
+- name: "Patrick Kalita"
+  position: "Software Engineer"
+  description: "Patrick is a software engineer with a focus on tools and systems for supporting scientific research and discovery. He has a BS from Rose-Hulman Institute of Technology (Electrical Engineering, 2006) and an MS from Purdue University (Electrical Engineering, 2008). Before joining Axiom Patrick’s work included web and data pipeline development for the Zebrafish Information Network (ZFIN) at the University of Oregon. Outside of work he paints and rides old bicycles."
+  image: "p-kalita"
+  email: "patrick@axiomdatascience.com"
+
```
Comment (srstsavage) [inline _data/staff.yml]: @patrick-axiom Looks good, but after the owners (top four) staff are in alphabetical order by last name. Can you move your section up?

**Review by srstsavage — CHANGES_REQUESTED** [review]
Move your section up to be in alphabetical order by last name. Otherwise looks good!

File: _data/staff.yml (hunk 1 above)
Comment (patrick-axiom) [reply _data/staff.yml]: Done!

**Review by srstsavage — APPROVED** [review]
LGTM!

**Pull request merged**